generator/main.go

```diff
@@ -95,6 +100,7 @@ func generateConfig(nodes *Node, nameToNode map[string]*Node, logger log.Logger)
 
 var (
 	failOnParseErrors  = kingpin.Flag("fail-on-parse-errors", "Exit with a non-zero status if there are MIB parsing errors").Default("false").Bool()
+	configPath         = kingpin.Flag("config-file", "Path to configuration file").Default("generator.yml").Short('c').String()
```

Comment:
We generally don't do short flags for things. We also prefix things so it's more clear what the flag is for.

```suggestion
	configPath         = kingpin.Flag("generator.config.file", "Path to configuration file").Default("generator.yml").String()
```